backend/.env.example

```
@@ -31,3 +31,9 @@ NAVER_CLIENT_SECRET=<from-1Password>
 
 # ─── Seoul Open Data ────────────────────────────────────────────
 SEOUL_API_KEY=<from-1Password>
+
+# ─── Auth (JWT + Google OAuth) ──────────────────────────────────
+JWT_SECRET=<from-1Password>
+JWT_ALGORITHM=HS256
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_ALGORITHM key should go before the JWT_SECRET key

(UnorderedKey)